exp/api/v1alpha3/awsmanagedmachinepool_types.go

```diff
@@ -73,6 +73,10 @@ type AWSManagedMachinePoolSpec struct {
 	// +optional
 	AdditionalTags infrav1alpha3.Tags `json:"additionalTags,omitempty"`
 
+	// AdditionalAsgTags is an optional set of tags to add to the underlying ASG resources managed by the AWS provider
+	// +optional
+	AdditionalAsgTags infrav1alpha3.Tags `json:"additionalAsgTags,omitempty"`
```

Comment:
Shall we rename it `AdditionalASGTags`. Same at other places also

Reply:
Sure, renamed to `AdditionalASGTags` :)